checkov/terraform/checks/resource/azure/ACREnableRetentionPolicy.py

```diff
@@ -10,8 +12,11 @@ def __init__(self) -> None:
         categories = (CheckCategories.GENERAL_SECURITY,)
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
-    def get_inspected_key(self) -> str:
-        return "retention_policy/enabled"
```

Comment:
Hi @paddymorgan84 thanks for the contribution! We still have users using azurerm version 3 (such as https://registry.terraform.io/providers/hashicorp/azurerm/3.117.0/docs/resources/container_registry), my recommendation is to modify the check to look for either attribute.

Reply:
Thanks for the feedback. Good point. I'll modify the policy to use `scan_resource_conf` instead and bolster the unit tests a little. I'm away for a few days but will look on my return 👍

Reply:
Thanks, @paddymorgan84! We got a support case about this one, so I'll go ahead and adjust it, but if you can fix your others, we'd really appreciate it. Enjoy the time away